TestData/EICAR.exe

```diff
@@ -1 +0,0 @@
-X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*
```

Comment:
Why did you remove this ?

Reply:
this was a test malware file which got cloned when I cloned the content repository. got deleted by AV  on my PC as it is a malware file